dotcom-rendering/src/components/MultiBylines.stories.tsx

```diff
@@ -0,0 +1,288 @@
+	body: [testTextElement],
+	byline: 'Guardian Contributor',
+	bylineHtml: "<a href='/profile/richard-hillgrove'>Richard Hillgrove</a>",
+	contributorIds: ['profile/richard-hillgrove'],
+	imageOverrideUrl:
+		'https://i.guim.co.uk/img/uploads/2024/09/17/Maurice_Casey.png?width=180&dpr=1&s=none',
+};
+
+const multiBylineWithNoByline = {
+	title: 'A further subheading',
+	body: [testTextElement],
+	endNote: 'This is an end note.',
+};
+
+const multiBylineWithNoContributorLink = {
+	title: 'This byline has a contributor with no link',
+	bio: testBioText,
+	body: [testTextElement],
+	byline: 'Steve McQueen on Paul Gilroy',
+	bylineHtml:
+		"<span data-contributor-rel='author'>Steve McQueen</span> on Paul Gilroy",
+};
+
+const args = (multiBylines: MultiByline[]) => ({
+	multiBylines,
+	tags: [
+		{
+			title: 'Richard Hillgrove',
+			id: 'profile/richard-hillgrove',
+			type: 'contributor',
+			bylineLargeImageUrl:
+				'https://i.guim.co.uk/img/static/sys-images/Guardian/Pix/pictures/2011/5/24/1306249890287/Richard-Hillgrove.jpg?width=100&dpr=2&s=none',
+		},
+	],
+	isLastElement: true,
+	/**
+	 * This will be replaced by the `formats` parameter, but it's
+	 * required by the type.
+	 */
+	format: {
+		design: ArticleDesign.Standard,
+		display: ArticleDisplay.Standard,
+		theme: Pillar.News,
+	},
+	abTests: {},
+	/**
+	 * This is used for rich links. An empty string isn't technically valid,
+	 * but there are no rich links in this example.
+	 */
+	ajaxUrl: '',
+	editionId: 'UK' as EditionId,
+	isAdFreeUser: false,
+	isSensitive: false,
+	pageId: 'testID',
+	switches: {},
+	RenderArticleElement,
+});
+
+export const ThemeVariations = {
+	args: args([
+		multiBylineWithLongHeader,
+		multiBylineWithImageOverride,
+		multiBylineWithNoByline,
+		multiBylineWithNoContributorLink,
+	]),
+	decorators: [centreColumnDecorator],
+	parameters: {
+		formats: getAllThemes({
+			design: ArticleDesign.Standard,
+			display: ArticleDisplay.Standard,
+		}),
+		chromatic: {
+			modes: {
+				horizontal: allModes.splitHorizontal,
+			},
+		},
+	},
+} satisfies Story;
+
+// Audio designs don't support multi-bylines
+const isNotAudioDesign = (format: ArticleFormat) =>
+	format.design !== ArticleDesign.Audio;
+
+export const DesignVariations = {
+	args: args([multiBylineWithLongHeader, multiBylineWithNoByline]),
+	decorators: [centreColumnDecorator],
+	parameters: {
+		formats: getAllDesigns({
+			theme: Pillar.News,
+			display: ArticleDisplay.Standard,
+		}).filter(isNotAudioDesign),
+		chromatic: {
+			modes: {
+				horizontal: allModes.splitHorizontal,
+			},
+		},
+	},
+} satisfies Story;
+
+export const DesignVariationsWithImageOverride = {
+	args: args([multiBylineWithImageOverride]),
+	decorators: [centreColumnDecorator],
+	parameters: {
+		formats: getAllDesigns({
+			theme: Pillar.News,
+			display: ArticleDisplay.Standard,
+		}).filter(isNotAudioDesign),
+		chromatic: {
+			modes: {
+				horizontal: allModes.splitHorizontal,
+			},
+		},
+	},
```

Comment:
I think this is identical to the story above (most likely because it's a small variation of the story above), so it could be replaced by?

```suggestion
	decorators: DesignVariations.decorators,
	parameters: DesignVariations.parameters,
```